apps/actions-dependencies-updater/src/caches.mts

```diff
@@ -0,0 +1,95 @@
+import * as log from "./logger.mjs";
+import { GithubShaToVersionCache } from "./github.mjs";
+import { UpdateTransaction } from "./updater.mjs";
+
+const CACHE_DIR = ".cache/";
```

Comment:
```suggestion
const CACHE_DIR = ".cache";
```

`join` will add path seperators for you

Reply:
That's what I thought too - but when I remove this slash I get this file in my package dir: `.cachesha-to-version.json`